GetandTake/Pages/Account/Login.cshtml.cs

```diff
@@ -0,0 +1,55 @@
+using Microsoft.AspNetCore.Authentication.Cookies;
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+using System.ComponentModel.DataAnnotations;
+using System.Security.Claims;
+
+namespace GetandTake.Pages.Account;
+
+public class LoginModel : PageModel
+{
+    [BindProperty]
+    [Required]
+    [Display(Name = "Email Address")]
+    public string EmailAddress { get; set; }
+
+    [BindProperty]
+    [Required]
+    [DataType(DataType.Password)]
+    public string Password { get; set; }
+
+    public IActionResult OnPost()
+    {
+        var isValidUser =
+               EmailAddress == "admin@cerensusuz.com"
+            && Password == "cerensusuzgetandtake!";
```

Comment:
Hardcoding in code your sensitive data is really a bad thing. If you want to use some predefined account data, let's put it into `appsettings.json`, **but** instead of putting there real values here you can just leave some example of data how it should look like